### PR TITLE
Add plot_compare_scenarios function to compare KPIs of chosen scenarios

### PR DESCRIPTION
**Pull request by MaGering**
With this PR it is possible to produce a plot for the comparison of the KPIs of chosen scenarios as seen in the figure below:

plot_scalars_E1_E2_E3_E4_E5_E6_F1_F2_F3_F4

The following steps were realized, as well (if applies):
- [ ] Use in-line comments to explain your code
- [ ] Write docstrings to your code
- [ ] For new functionalities: Explain in readthedocs
- [ ] Write test(s) for your new patch of code
- :x: Update the CHANGELOG.md (let's start this after the first release)
- [ ] Apply black (`black . --exclude docs/`)
- [ ] Check if full simulation tests pass locally (`EXECUTE_TESTS_ON=master pytest`)

*Please mark above checkboxes as following:*
- [ ] Open
- [x] Done

:x: Check not applicable to this PR

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl-institut/mvs_eland/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

**Comment by MaGering** [comment]
@Piranias could you please help me adjusting the limits of the y-axis for the KPI `Installed capacity PV`? The subplot is cut off (see image above). I have tried a lot but did not succeed in fixing it. One idea I had was scaling the axis down to `x10⁴` but this didn't fix it either (the plot has remained the same).

**Comment by Piranias** [comment]
> @Piranias could you please help me adjusting the limits of the y-axis for the KPI `Installed capacity PV`? The subplot is cut off (see image above). I have tried a lot but did not succeed in fixing it. One idea I had was scaling the axis down to `x10⁴` but this didn't fix it either (the plot has remained the same).

ok, i'll take a look. Looks great already! :)

File: pvcompare/plots.py [hunk 1]
```diff
@@ -821,6 +817,219 @@ def plot_facades(
...
+    handles, labels = ax.get_legend_handles_labels()
+    fig.legend(
+        handles, labels, loc="lower left", mode="expand",
+    )
+
+    name = ""
+    for scenario_name in scenario_name_ending:
+        name = name + "_" + str(scenario_name)
+
+    fig.savefig(os.path.join(outputs_directory, "plot_scalars" + str(name) + ".png",))
```
Comment (Piranias) [inline pvcompare/plots.py:1030]: could you change the plot name from "plot_scalars" to something else? So we do not get confused with the plot_KPI_loop outputs

Comment (MaGering) [reply pvcompare/plots.py:1030]: Oh of course! Thank you.

Comment (MaGering) [reply pvcompare/plots.py]: Done with commit https://github.com/greco-project/pvcompare/pull/243/commits/8d1ff4e6f9ab7fe16c40c688e69eacb5371c1561.

**Pull request merged**